src/pages/docs/security/users-and-teams/auditing/index.md

```diff
@@ -66,9 +66,7 @@ We take the sensitive value and hash it using an irreversible hash algorithm. We
 ## Archived audit logs {#archived-audit-events}
 
 :::div{.hint}
-The audit log retention functionality was gradually made available to **Cloud** customers from **Octopus 2023.1** onwards.
-
-Audit log retention will be made available to self-hosted customers later in 2023. However, from **Octopus 2022.3** onwards, self-hosted customers will be able to change the **Event retention days** configuration, allowing you to adjust to your preferred retention period prior to retention archiving audit logs. 
+The audit log retention functionality is available from **Octopus 2023.3** onwards.
```

Comment:
Having done the fix to RPC retries, I was reminded that we do similar version distinctions in event retention.

Applying the same logic, I felt that it has been around for a while now, and didn't need as much fan-fair now.

Reply:
I'm pretty sure this became available to the public in `2023.3`. [The feature toggle was removed then](https://github.com/OctopusDeploy/OctopusDeploy/commit/229ab39e085096650991e14f947506e983360b43), so I believe that's good evidence 😁

Reply:
LGTM